wandb/sdk_py27/wandb_run.py

```diff
@@ -667,8 +667,7 @@ def log_code(
         root = ".",
         name = None,
         include_fn = lambda path: path.endswith(".py"),
-        exclude_fn = lambda path: os.sep + "wandb" + os.sep
-        in path,
+        exclude_fn = filenames.exclude_wandb_fn,
```

Comment:
oddly when i left this in a lambda, mypy had trouble parsing so i moved it into a function.